themes/finna2/templates/librarycards/connectedusers.phtml

```diff
@@ -0,0 +1,24 @@
+<h2 class="lightbox-header"><?=$this->transEsc('library_card_connected_users'); ?></h2>
+<table class="table table-striped">
+  <tr>
+    <th><?=$this->transEsc('library_card_date_of_connection')?></th>
+    <th><?=$this->transEsc('Institution')?></th>
+    <th><?=$this->transEsc('account_login_method')?></th>
+  </tr>
+  <?php foreach ($this->users as $user): ?>
+    <tr>
+      <td><?= $this->escapeHtml($this->dateTime()->convertToDisplayDateAndTime('Y-m-d H:i:s', $user['cardCreated']))?></td>
+      <td>
+        <?php if ($user['mainView']): ?>
+          <?php [$inst, $view] = explode('/', $user['mainView']); ?>
```

Comment:
Siltä varalta, että mainView-asetus on pielessä, pitää tehdä näin:
`          <?php [$inst, $view] = explode('/', $user['mainView'] . '/default'); ?>`